.github/workflows/publish.yml

```
@@ -5,33 +5,47 @@ on:
+        uses: actions/setup-node@v6
+
+      - name: Install npm dependencies
+        run: yarn install --frozen-lockfile
+
+      - name: Release
+        id: release
+        env:
+          API_TOKEN_GITHUB: ${{ secrets.API_TOKEN_GITHUB }}
+          GITHUB_TOKEN: ${{ secrets.API_TOKEN_GITHUB }}
+          GITHUB_USER: ${{ github.actor }}
+        run: yarn run semantic-release --extends ./release.config.${{ matrix.job.arch }}.js --tag-format='${version}-${{ matrix.job.arch }}'
```

actionlint (1.7.12)

[error] property "job" is not defined in object type {}

(expression)